src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+
+const IncludeGrid = styled(GridItemConstrainedMedium)`
+  display: grid;
+`;
+
+// currently ampSrc: 'http://localhost:7080 need to replace start of this with // https://news.test.files.bbci.co.uk
+
+const VjAmp = ({ ampSrc, ampImage, ampImageHeight, ampImageWidth }) => {
+  return (
+    <IncludeGrid>
+      <amp-iframe
+        width={ampImageWidth}
+        height={ampImageHeight}
+        layout="responsive"
+        sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms"
```

Comment:
How did you decide to use these attributes?
- `allow-scripts` - can see obviously why we need this
- `allow-same-origin` - not sure specifically why we need this?
Using both together is discouraged here:
> <img width="668" alt="image" src="https://user-images.githubusercontent.com/9645462/85590745-91491400-b63c-11ea-8d0e-ca0ca18a7f8f.png">
- `allow-top-navigation-by-user-activation` looks like that is needed to navigate to another BBC page off an include I can forsee that being needed
- `allow-forms` - can imagine this may be needed for user interactions like quizzes

They are all explained here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe 

@eagerterrier can you shed any light on which of these sandbox flags we need?

Reply:
Thanks for the link 👍 

> How did you decide to use these attributes?

I copied the implementation that is used in Alephant.
Then found that without `allow-same-origin` the include does not load properly and displays the fallback saying javascript is needed.

In the Notes about sandboxing, it says that,
'When the embedded document has the same origin as the embedding page, it is strongly discouraged to use both `allow-scripts` and `allow-same-origin`'

Doesn't the embedded document have a different origin to the embedding page here? 


> `allow-same-origin` - not sure specifically why we need this?

If there is no `allow-same-origin` then from [here:](https://github.com/ampproject/amphtml/blob/master/spec/amp-iframe-origin-policy.md) 'If one does not opt into allow-same-origin, then every origin is allowed for the iframe' 

Would this be a security risk to not include it?